torpaste.py

```diff
@@ -43,6 +47,7 @@ def new_paste():
         if (request.form['content']):
             status, message = logic.create_new_paste(
                 request.form['content'],
+                request.form['visibility'],
```

Comment:
Similarly, I'm not sure whether this works when the unlisted paste checkbox is missing because it's disabled. Maybe an `or 'public'` will do? But I'm not sure since `Flask` may catch the `KeyError` exception and respond manually..